build.sh

```
@@ -1,4 +1,66 @@
-pip install fasttext && \
-poetry install --without dev --all-extras  && \
-poetry build && \
+echo "Running build.sh"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)